src/datasources/entity-ds-react/EntityDataSource.ts

```diff
@@ -60,6 +78,16 @@ export class EntityDataSource extends DataSourceApi<EntityQuery> {
     }
 
     async metricFindQuery(query, options) {
+        // TODO: Might be good to be able to pass in options.entityType + options.attribute
```

Comment:
I'm a fan of capturing TODOs in tickets only so that we don't forget. So easy to lose TODOs like this otherwise.